frontend/src/main/components/Jobs/InstructorReportJobForm.js

```diff
@@ -0,0 +1,60 @@
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+function InstructorReportJobForm({ submitAction }) {
+
+ const defaultValues = {
+    fail: false,
+    sleepMs: 1000
+};
```

Comment:
These values are not needed for the Instructor Job Form.   They are only needed for the TestJob.

I'd suggest making an empty defaultValues object though in case there are some parameters to this job later on.  (For now, there aren't any).